.devcontainer/typescript/devcontainer.json

```
@@ -0,0 +1,24 @@
+{
+	"name": "TypeScript",
+	"dockerComposeFile": [
+	  "./docker-compose.yml"
+	],
+	"workspaceFolder": "../typescript",
```

workspaceFolder should be absolute path.

```suggestion
	"workspaceFolder": "/workspaces/typescript",
```